2d-survival-template/project/enemy/enemy.gd

```diff
@@ -1,6 +1,6 @@
 extends StaticBody2D
 
-@onready var player: CharacterBody2D = %Player
+@export var player: CharacterBody2D
```

Comment:
For the sake of faster config, I think either having the enemy find the player themself with g_first_node_in_group or having also a default value of %Player would be ideal.

When we create a new scene (like a test or one to try a new level layout), Adding a player and an enemy is not plug and play, the enemy does not work out of the box and must be configured.

Reply:
I agree.

Reply:
0 config, default behavior added: https://github.com/loteque/2024_oh_jam/pull/2/commits/fea2f75a25268a8b315f0e973845e40823d845fb